### ISSUE TITLE
Detector pixel coordinate specification isn't necessarily consistent with odd/even array sizes

### ISSUE THREAD

**Issue by mperrin** [spacetelescope/webbpsf#200]
<a href="https://github.com/mperrin"><img src="https://avatars0.githubusercontent.com/u/1151745?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [mperrin](https://github.com/mperrin)**
_Friday Apr 06, 2018 at 20:13 GMT_
_Originally opened as https://github.com/mperrin/webbpsf/issues/200_

----

As discussed with @shanosborne 

Consider:
```
fgs = FGS()
fgs.detector_position= (100,400)
psf = fgs.calc_psf(fov_pixels=100)
```
The resulting PSF has even dimensions so it's exact center is implicitly on a half pixel boundary. The detector position coordinates written into the FITS header should probably be modified to reflect that. 

Could also consider giving the mapping to the detector pixel coordinates of the first pixel (data[0,0]) in the array?  Need to be more thoughtful about the precise consistency of this with the SIAF values.

**Comment by mperrin** [comment]
<a href="https://github.com/mperrin"><img src="https://avatars0.githubusercontent.com/u/1151745?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [mperrin](https://github.com/mperrin)**
_Wednesday Apr 25, 2018 at 17:03 GMT_

----

@shanosborne is fixing this

**Comment by mperrin** [comment]
<a href="https://github.com/mperrin"><img src="https://avatars0.githubusercontent.com/u/1151745?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [mperrin](https://github.com/mperrin)**
_Friday Apr 27, 2018 at 14:11 GMT_

----

Fixed by #205
